config/detekt.yml

```diff
@@ -5,6 +5,7 @@ test-pattern: # Configure exclusions for test sources
   active: true
   patterns: # Test file regexes
     - '.*/test/.*'
+    - '.*/androidTest/.*'
```

Comment:
Why do we exclude detekt here?

Reply:
We were following the line above. Does the project want static analysis on UI Tests?

Reply:
Mh, okay. Then let's land it for now. In general I think it would be good to run it on that code too just to avoid common mistakes and have a unified formatting. Anyhow, no need to block landing.